src/display/editor/outliner.js

```diff
@@ -301,13 +305,21 @@ class HighlightOutline extends Outline {
     return buffer.join(" ");
   }
 
-  serialize(x, y, width, height) {
+  /**
+   * Serialize the outlines into the PDF page coordinate system.
+   * @param {Array<number>} param0 - the bounding box of the annotation.
```

Comment:
Can't we use a consistent parameter name instead here?
```suggestion
   * @param {Array<number>} _bbox - the bounding box of the annotation.
```